edgedb/_cluster.py

```diff
@@ -95,10 +95,7 @@ def __init__(
 
     @staticmethod
     def get_edgedb_server():
-        try:
-            return f'edgedb-server-{os.environ["EDGEDB_CI_SLOT"]}'
-        except KeyError:
-            return 'edgedb-server'
+        return 'edgedb-server'
```

Comment:
What's changed here? We still need binary with a suffix.

Reply:
No, `edgedb-setup` adds the directory with unsuffixed `edgedb-server` to `PATH`.